src/basedtyping/transformer.py

```diff
@@ -0,0 +1,231 @@
+from __future__ import annotations
+
+import ast
+import types
+import typing
+import uuid
+from contextlib import contextmanager
+from enum import Enum
+from typing import cast
+
+import typing_extensions
+
+import basedtyping
+
+
+# ruff: noqa: N802, S101
+class CringeTransformer(ast.NodeTransformer):
+    """Transforms `1 | 2` into `Literal[1] | Literal[2]` etc"""
```

Comment:
shouldnt it be this instead
```suggestion
    """Transforms `1 | 2` into `Literal[1, 2]` etc"""
```

Reply:
welll, the actual impl does create the `Literal[1] | Literal[2]`, they are equivalent though